app/data_model/answer_store.py

```diff
@@ -247,6 +254,26 @@ def get_hash(self):
         """
         return hash(json.dumps(self.answers, sort_keys=True))
 
+    def upgrade(self):
+        if self.version < self._LATEST_VERSION:
+            if self.version < 1:
```

Comment:
Can we split this out to make it clearer that upgrade is a generic change to the answer store structure and the second "if" is for the specific date upgrade.